src/metabase/api/table.clj

```diff
@@ -316,5 +317,9 @@
     (db/simple-delete! FieldValues :id [:in field-ids]))
   {:status :success})
 
+(api/defendpoint GET "/:id/related"
+  "Return related entities."
```

Comment:
Are there any tests for this, or the other related entities, endpoints?

Reply:
Kind of -- the last test in related-test goes through the API endpoint. But I can be easily persuaded to do an explicit API endpoint one as well.

Reply:
Added tests for all API endpoints.